src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public String getRandomBall () {
+        Ball ball = new Ball();
+        ball.color = new ColorSupplier().getRandomColor().toString();
+        ball.number = new Random().nextInt(100);
```

Comment:
`ColorSupplier` and `Random` should be class-level variables
[All magic numbers in your code should be constants.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants)